doc/source/getting_started/earthdata.rst

```diff
@@ -0,0 +1,40 @@
+=====
```

Comment:
I'm excited by the addition of this documentation to help users get set up and oriented and using their Earthdata login within icepyx (I just realized the other day that all of our password storage options aren't well documented, so I wish I'd been able to tackle this PR sooner). I'm wondering if we should give it a broader name (e.g. data access) with multiple sections: one focused on Earthdata login options (which could (should?) also be added to the API docstrings); other data access examples (though we've also got some links to these types of things on the in-need-of-an-update [ICESat-2 resources](https://icepyx.readthedocs.io/en/latest/community/resources.html#other-ways-to-access-icesat-2-data); and common points of confusion specific to ICESat-2 data access (CMR metadata search versus subsetting returns --> and the NSIDC programmatic access guide has a great visual to help explain this).

Reply:
these are great points and great ideas.  Happy to help get these docs going :+1:

Reply:
I'm thinking about moving this effort to its own branch - any objections?

Reply:
Sounds good :+1:

Reply:
Finally created a new branch for this - login_docs.